docs/docs/reference/attestations.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_position: 1
+title: Contract-less and auto-discovery of materials
+---
+A Workflow Contract specifies the necessary content that a workflow must include in its attestation. For instance, it might mandate 
+the inclusion of the URI@digest of the generated container image, the container root filesystem used during the build, and the 
+Software Bill of Materials for that container image. These pieces of evidence must be associated with a specific material type. 
+Operators define what must be included in the contracts, and developers need to understand and comply with these requirements.
+
+This has been the case up until now, with the introduction of contract-less materials and auto-discovery, we ease the job of operators 
+and developers when working with attestations.
+
+## Contract-less materials
+Not all pieces of evidences need to to be registered as a material on the contract, you can add as many pieces of evidences as you like by adding 
+its value and a new flag `--kind`, which determines that type of material you’re attesting, example:
+
+```bash
+$ chainloop attestation init --workflow-name wf-test
```

Comment:
please replace `workflow-name` for `name`

Reply:
Good catch, will do!